src/main/java/ua/javarush/island/map/Location.java

```diff
@@ -0,0 +1,102 @@
+package ua.javarush.island.map;
+
+import ua.javarush.island.actions.Eatable;
+import ua.javarush.island.actions.Movable;
+import ua.javarush.island.actions.Reproducible;
+import ua.javarush.island.entities.Entity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class Location implements Reproducible, Eatable, Movable {
```

Comment:
Why does location implement interfaces Reproducible, Eatable, Movable?
Do we expect the location to move or something like that?